package.json

```diff
@@ -84,7 +84,7 @@
     "jquery-ui": "https://jqueryui.com/resources/download/jquery-ui-1.12.1.zip",
     "jquery-ui-themes": "https://jqueryui.com/resources/download/jquery-ui-themes-1.12.1.zip",
     "summernote-nugget": "https://github.com/pHAlkaline/summernote-plugins",
-    "literallycanvas": "https://github.com/literallycanvas/literallycanvas/archive/v0.4.13.tar.gz",
+    "literallycanvas": "https://github.com/literallycanvas/literallycanvas/archive/v0.4.14.tar.gz",
```

Comment:
Are we sure that this works? I think I recall that version missing files.

Reply:
Missing files are because jquery plugin interface was removed. Lib was broke anyway just not sure when. I've been playing with because I want to add a dialog that allows diagram selection and text templates.
This is where somebody tells me to quit developing new features and work on BS but, simply can't help myself!